vagrant/Vagrantfile

```diff
@@ -107,8 +55,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # Default value: false
         target.ssh.forward_agent = true
 
-        target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
-        target.vm.provision "file", source: "./blacklist_mongo.txt", destination: "blacklist_mongo.txt"
+        #target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
```

Comment:
not big concern. if you got solid answer if those 2 lines are no longer useful per US team feedback later.
you can remove them and the files in on-build-config/vagrant/